packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -11,12 +11,16 @@ class SimpleExpansionPanelListTestWidget extends StatefulWidget {
     this.firstPanelKey,
     this.secondPanelKey,
     this.canTapOnHeader = false,
+    this.expandedHeaderPadding,
```

Comment:
Set the default value here to:
```
EdgeInsets.symmetric(
    vertical: 64.0 - _kPanelHeaderCollapsedHeight
);
```

That way, you won't have to create the null check control flow for `widget.expandedHeaderPadding`, which can be confusing.

Since both are private properties in `expansion_panel.dart`, you could recreate the constants in this test file.

Reply:
We could recreate the constants in the test, but tests for the default padding would suffer then as
* No usage of SimpleExpansionPanelListTestWidget possible
* Only tests for inital state possible, as SimpleExpansionPanelListTestWidget is needed for triggering the header tap (setState()) call.

What about using the following hack `const ExpansionPanelList().expandedHeaderPadding` as introduced in the latest change?